app/src/main/res/drawable/item_border.xml

```diff
@@ -0,0 +1,11 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="@color/white" />
+    <stroke
+        android:width="1dp"
+        android:color="@color/lightGrey" />
+    <padding
+        android:left="4dp"
```

Comment:
Missed the dimens for these

Reply:
These dimensions will be refactored into `dimen.xml`.